custom/enikshay/integrations/bets/repeaters.py

```diff
@@ -291,7 +295,22 @@ def allowed_to_forward(self, episode_case):
         )
 
 
-class BETSLocationRepeater(LocationRepeater):
+class BETSUserRepeater(BETSRepeaterMixin, UserRepeater):
+    friendly_name = _("Forward users to BETS")
+    payload_generator_classes = (UserPayloadGenerator,)
+
+    location_types_to_forward = ['plc', 'pcp', 'pcc', 'pac']
+
+    def _is_relevant_location(self, location):
+        return (location.metadata.get('is_test') != "yes"
+                and location.location_type.code in self.location_types_to_forward)
```

Comment:
c'mon stickler: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

>  Donald Knuth explains the traditional rule in his Computers and Typesetting series: "Although formulas within a paragraph always break after binary operations and relations, displayed formulas always break before binary operations" ... In Python code, it is permissible to break before or after a binary operator, as long as the convention is consistent locally. For new code Knuth's style is suggested.